rollingpin/status.py

```diff
@@ -64,20 +64,14 @@ def fetch_deploy_status(config):
 
     fetch_req = _do_status_http_request(harold_base_url, harold_secret, salon)
```

Comment:
i tried to unify this into the haroldwhisperer client but it looks like the signature algorithm is different between the two so I didn't wanna pollute the PR